core/trino-web-ui/src/main/resources/webapp-preview/eslint.config.js

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Can we extract this to a separate change?

Reply:
removed from this PR